stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        predicted_results = model.predict(predictors)
</pre><p> </p><p>Maybe give all functions at least a short docstring to describe what they do (I only noticed because the next function has such a full one, I don't think they all need to be like that but at least a line summarising what the function does).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Done, except for a few small ones that seemed abundantly self-evident.</p>